site/src/pages/DeploymentSettingsPage/GeneralSettingsPage/GeneralSettingsPageView.tsx

```diff
@@ -36,6 +37,12 @@ export const GeneralSettingsPageView: FC<GeneralSettingsPageViewProps> = ({
 	safeExperiments,
 	invalidExperiments,
 }) => {
+	const licenseUtilizationPercentage =
+		entitlements?.features?.user_limit?.actual &&
+		entitlements?.features?.user_limit?.limit
+			? entitlements.features.user_limit.actual /
+				entitlements.features.user_limit.limit
+			: undefined;
```

Comment:
Suggestion: Instead of just doing the percentage here, we could prepare the data format in a more convenient way to use in the template:

```typescript
	const licenseLimit =
		entitlements?.features?.user_limit?.actual &&
		entitlements?.features?.user_limit?.limit
			? {
				valid: true,
				actual: entitlements.features.user_limit.actual,
				limit: entitlements.features.user_limit.limit
			} : {
				valid: false,
				actual: 0,
				limit: 0,
			};
```

Might not need to even init the actual/limit for the zero case but added it just in case.